ycmd.el

```diff
@@ -923,7 +923,7 @@ nil, this uses the current buffer.
                          (`force-semantic '(("force_semantic" . "true")))
                          (`tags (-when-let (tag-files (ycmd--get-tag-files buffer))
                                   `(("tag_files" . ,tag-files)))))))
-    (append standard-content extra-content)))
+    (append extra-content standard-content)))
```

Comment:
Minor optimization here because performance is better when appending to front instead of back